### PR TITLE
[stable/rbac-manager]  support image digest

### PR DESCRIPTION
**Pull request by swarren83**
**Why This PR?**
Add support for image digest used by cosign to verify signed images.

**Changes**
Changes proposed in this pull request:

* support image digest

**Checklist:**

* [x] I have included the name of the chart in the title of this PR in square brackets i.e. `[stable/goldilocks]`.
* [x] I have updated the chart version in `Chart.yaml` following Semantic Versioning.
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] Any new values have been added to the README for the Chart, or `helm-docs --sort-values-order=file` has been run for the charts that support it.

### CODE REVIEW

**Comment by swarren83** [comment]
Typically cosign will using a mutating webhook to cross reference tags with the digest to verify the image the tag is for is signed.  In our use case we have ArgoCD that manages resources and the mutating webhook and ArgoCD fight over the state of the resource.  This requires us to use the sha256 digest versus a tag.

**Review by sudermanjr — CHANGES_REQUESTED** [review]
Thanks for the PR! I have always kind of thought we should just start using digest more frequently everywhere :-D 

I'm thinking it might be good to add a file called `ci/digest-values.yaml` and then put an image.digest in that file, so that we run an e2e test using this new value.

**Comment by swarren83** [comment]
@sudermanjr Thank you for the prompt response.  I have added the suggested file.

File: stable/rbac-manager/ci/digest-values.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+image:
+  digest: sha256testvalue
```
Comment (sudermanjr) [inline stable/rbac-manager/ci/digest-values.yaml]: Linter apparently doesn't like this lack of newline

Comment (sudermanjr) [reply stable/rbac-manager/ci/digest-values.yaml]: Oh, and I think we will need to put a valid sha256 for the end to end tests to pass here.

Comment (sudermanjr) [inline stable/rbac-manager/ci/digest-values.yaml]:
```suggestion
  digest: sha256:a2f3d6fb37a22e2ecdb87d8f4f7984c639fa29955cb7e43400ca32804a5c2a2c

```

**Comment by swarren83** [comment]
> Oh, and I think we will need to put a valid sha256 for the end to end tests to pass here. 

I added the v1.4.0 digest sha and the newline.

**Review by sudermanjr — APPROVED** [review]
Thanks for the quick fixes!

**Comment by swarren83** [comment]
@sudemanjr Thank you for the quick updates.  Look forward to submit more PRs in the future.

**Pull request merged**